README.md

```diff
@@ -76,6 +76,36 @@ Most of the bindings ([`src/auto`](src/auto)) are generated by [gir](https://git
 
 When opening a PR please put the changes to the `src/auto` directory in a separate commit.
 
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/gtk-rs/gtk/graphs/contributors"><img src="https://opencollective.com/gtk-rs/contributors.svg?width=890&button=false" /></a>
+
+### Financial Contributors
+
+Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/gtk-rs/contribute)]
+
+#### Individuals
+
+<a href="https://opencollective.com/gtk-rs"><img src="https://opencollective.com/gtk-rs/individuals.svg?width=890"></a>
+
+#### Organizations
+
+Support this project with your organization. Your logo will show up here with a link to your website. [[Contribute](https://opencollective.com/gtk-rs/contribute)]
+
+<a href="https://opencollective.com/gtk-rs/organization/0/website"><img src="https://opencollective.com/gtk-rs/organization/0/avatar.svg"></a>
```

Comment:
What happened here? O.o

Reply:
I may have hit return?

Reply:
What do you mean? These are placeholders to allow multiple sponsors.